### ISSUE TITLE
problem with StringUtils 

### ISSUE THREAD

**Issue by TrejGun** [tonivj5/typeorm-naming-strategies#22]
Hi there! I have a typescript project with 
typeorm 0.2.44 and typeorm-naming-strategies 3.0.0

this is what I see on startup

```
node:internal/modules/cjs/loader:488
      throw e;
      ^

Error: Cannot find module '~/node_modules/typeorm/util/StringUtils.js.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:960:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:953:15)
    at resolveExports (node:internal/modules/cjs/loader:482:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (~/node_modules/typeorm-naming-strategies/snake-naming.strategy.js:5:26)
    at Module._compile (node:internal/modules/cjs/loader:1101:14) {
  code: 'MODULE_NOT_FOUND',
  path: '~/node_modules/typeorm/package.json'
}
```

I believe this is related to typescript config but I have to idea how it should look like for this to work

**Comment by segomin** [comment]
I just remove `.js` at `node_module/typeorm-naming-strategies/snake-naming.strategy.js` as `const StringUtils_js_1 = require("typeorm/util/StringUtils");` for temporally. But this is just for local running. 
I should copy the whole library and override it for deploy unless finding out the right solution.

**Comment by segomin** [comment]
btw, "typeorm-naming-strategies": "^2.0.0" works fine for me.

**Comment by JaehyeonLim** [comment]
I had the same problem, so I rolled back to version 2.0.

**Comment by tonivj5** [comment]
It's related to https://github.com/tonivj5/typeorm-naming-strategies/pull/15 (there is the long story)

I've merged #23, which resolve this issue, I'll publish a new release (`4.0`). By the moment, you can use `2.0`, it's exactly equal to `3.0` (without the "bug")

**Comment by jpagex** [comment]
@tonivj5 When do you plan to release the fix (`4.0`)? It seems to fix the issue for me.

**Comment by tonivj5** [comment]
Hey @jpagex and #all `4.0.0` is out https://github.com/tonivj5/typeorm-naming-strategies/releases/tag/4.0.0, as said before it's the same than `2.0.0`
